test/acceptance/features/steps/steps.py

```diff
@@ -145,26 +152,35 @@ def db_instance_is_running(context, db_name):
 @when(sbr_is_applied_step)
 def sbr_is_applied(context):
     sbr_yaml = context.text
-    sbr = ServiceBindingRequest()
-    if context.__contains__("nodejs_app"):
-        application = context.nodejs_app
-        context.nodejs_app_original_generation = application.get_observed_generation()
-        context.nodejs_app_original_pod_name = application.get_running_pod_name()
+    namespace = context.namespace
+    sbr = ServiceBindingRequest(namespace.name)
```

Comment:
cannot we have simply 

```
 sbr = ServiceBindingRequest(context.namespace)

```

Reply:
` sbr = ServiceBindingRequest(context.namespace)` will not fetch the name of the namespace. i think it gets namespace object. maybe we should do ` sbr = ServiceBindingRequest(context.namespace.name)`. segregated for better readability